src/android/CameraLauncher.java

```diff
@@ -270,18 +279,20 @@ public void getImage(int srcType, int returnType, int encodingType) {
         croppedUri = null;
         if (this.mediaType == PICTURE) {
             intent.setType("image/*");
-            if (this.allowEdit) {
+            if (this.onlyGalleries) {
```

Comment:
you're probably right, but since the allowEdit option was already forcing the galleries only option, I thought this way would have the least impact